src/pages/app-bar/contextual-action/index.tsx

```diff
@@ -0,0 +1,350 @@
+import React, { useCallback, useState } from 'react';
+import {
+    AppBar,
+    Button,
+    Checkbox,
+    Hidden,
+    IconButton,
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+    Toolbar,
+    Typography,
+    useMediaQuery,
+} from '@material-ui/core';
+import { makeStyles, Theme, useTheme } from '@material-ui/core/styles';
+import CloseIcon from '@material-ui/icons/Close';
+import MenuIcon from '@material-ui/icons/Menu';
+import DeleteIcon from '@material-ui/icons/Delete';
+import { Spacer } from '@pxblue/react-components';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import * as colors from '@pxblue/colors';
+import clsx from 'clsx';
+
+export type ListItemType = {
+    id: number;
+    name: string;
+    ip: string;
+    checked: boolean;
+};
+
+const useStyles = makeStyles((theme: Theme) => ({
+    appbar: {
+        transition: theme.transitions.create('opacity', { duration: theme.transitions.duration.standard }),
+    },
+    appbarRoot: {
+        padding: 0,
+    },
+    contextualAppBar: {
+        backgroundColor: colors.black[500],
+        color: colors.white[50],
+        right: 0,
+        width: 0,
+        '&$contextualBarActive': {
+            [theme.breakpoints.down('sm')]: {
+                width: '100%',
+            },
+        },
+    },
+    contextualBarActive: {},
+    checkboxCell: {
+        padding: `0 0 0 ${theme.spacing(2.5)}px`,
+        minWidth: '72px',
+        [theme.breakpoints.down('sm')]: {
```

Comment:
This rule may have caused some alignment issue

![image](https://user-images.githubusercontent.com/8997218/134526262-7a3218d5-ada1-45ae-a688-6c8892e65954.png)